2_Growth-Maturity-Decline.md

```diff
@@ -3,75 +3,26 @@
 
  Activity Metric | Description
  --- | ---
- Tickets Closed | Total number of tickets closed
- Time to Close | Time from bug opened to bug closed
- Time to Merge | Time from submission to merge or abandon
- Review Efficiency | Number of merged patches / Number of abandoned patches
+ Tickets Closed / Tickets Opened | Total number of tickets closed per tickets opened over time
+ Bug Time to Close | Time from bug opened to bug closed over time
+ Time to Merge | Time from submission to merge or abandon over time
+ Review Efficiency | Number of merged patches / Number of abandoned patches over time
  Number of New Contributors | Number of first time contributors over time
  Community Growth | Number of new developers over time
 
+**Disclaimer:**
+The activity metrics listed are not meant to represent a fully comprehensive list. It is fully expected that this list will evolve as people have insights and thoughts about the activity metrics that comprise Growth-Maturity-Decline. 
 
-**Background:**
-At the Open Source Summit North America, a group of around 8-12 people each agreed on the metrics important in the following scenarios.
-For now, we will focus our conversation around these outcomes to further the conversation.
-The two-hour session sparked great conversations but could not address all aspects.
-We thank everyone who participated.
+**Tooling:**
+The activity metrics are intended to be a starting point for community health related tooling. It is expected that the activity metrics will evolve based on the ability (or inability) of tooling to successfully implement the activity metrics. 
 
-Below is the metric->signal->action chain for *Growth-Maturity-Decline*.
+**Background:**
+The activity metics have been identified based on workshops at the Open Source Leadership and the Open Source Summit North America. In addition, the activity metrics are based on active CHAOSS mailing list conversations. The activity metrics listed here are the result of compiling the discussions to data. We thank everyone who participated.
 
 **How to contribute:**

```

Comment:
Maybe it would be better to have a link to a "contributing.md" page, which would have the text in all these pages about contribution? That way we have a single point for maintaining info on contributions.

Reply:
Good idea. The reason that I included the background was to help explicate that these metrics weren't just made up :) That we have been spending time -- and will continue to spend time on what the metrics can be.